nx.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": "nx/presets/npm.json",
+    "tasksRunnerOptions": {
+        "default": {
+            "runner": "nx/tasks-runners/default",
+            "options": {
+                "cacheDirectory": "./.nx-cache",
+                "cacheableOperations": ["prepare", "build", "dev"],
+                "parallel": 20
```

Comment:
I'm really surprised that NX [doesn't default](https://nx.dev/nx/affected#parallel) to the number of available cores on the machine. It seems odd to me to hard-code it.

Reply:
I was surprised they don't do this by default either.  Interestingly, when I was tinkering with the [useNx](https://github.com/lerna/lerna/blob/1c358286c8425d1720580859c4b42e15a3b15eac/commands/run/README.md#usenx-experimental) feature in Lerna it looks like the [parallelism](https://github.com/lerna/lerna/blob/34286f221eec18db3383daddd0f615baca6e7510/commands/run/index.js#L203-L213) in nx is set to [the number of packages in the repository](https://github.com/lerna/lerna/blob/34286f221eec18db3383daddd0f615baca6e7510/commands/run/index.js#L241-L245).

I'm testing it out with nucleus to see if it can handle the concurrency but if it can't we only really need this much parallelism for the `dev` command.

I think if nucleus can't handle the level of parallelism I may default the value back to 3 and use the --parallel=20 argument for `yarn dev`.